src/components/Accordion/Root.tsx

```diff
@@ -1,29 +1,48 @@
-import { Root as AccordionRoot } from "@radix-ui/react-accordion";
+import {
+  AccordionMultipleProps,
+  Root as AccordionRoot,
+  AccordionSingleProps,
+} from "@radix-ui/react-accordion";
 import { forwardRef, ReactNode } from "react";
 import { Box, PropsWithBox } from "../Box";
 
 export type AccordionRootProps = PropsWithBox<{
   children: ReactNode;
-  defaultValue?: string;
-  value?: string;
-  onValueChange?: (value: string) => void;
+  type?: "single" | "multiple";
+  defaultValue?: string | string[];
+  value?: string | string[];
+  onValueChange?: (value: string | string[]) => void;
 }>;
```

Comment:
IMO would be simpler to have union here and you will not need checking types manually
```suggestion
export type AccordionRootProps = PropsWithBox<{type: "singe", defaultValue: string ...} | {type:"multiple", defaultValue:string[] ....}>
```